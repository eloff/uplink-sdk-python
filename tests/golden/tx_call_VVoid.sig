r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 47547086946621824966968396536816218704665375210038806737081552562638733515775
