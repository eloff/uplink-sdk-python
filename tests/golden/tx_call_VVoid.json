{
    "header": {
        "contents": {
            "contents": {
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
                "args": [
                    {
                        "tag": "VVoid"
                    }
                ], 
                "method": "get"
            }, 
            "tag": "Call"
        }, 
        "tag": "TxContract"
    }, 
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE00NzU0NzA4Njk0NjYyMTgyNDk2Njk2ODM5NjUzNjgxNjIxODcwNDY2NTM3NTIxMDAzODgwNjczNzA4MTU1MjU2MjYzODczMzUxNTc3NQ=="
}